include/zenoh-pico/api/types.h

```diff
@@ -547,16 +547,16 @@ typedef struct {
  *
  * Members:
  *   void *context: a pointer to an arbitrary state.
- *   z_reply_handler_t call: `void (*_z_reply_handler_t)(_z_reply_t *reply, void *arg)` is the
+ *   z_reply_handler_t call: `void (*_z_reply_handler_t)(z_loaned_reply_t *reply, void *arg)` is the
  * callback function.
  *   z_dropper_handler_t drop: `void *drop(void*)` allows the callback's state to be freed.
  *   void *context: a pointer to an arbitrary state.
  */
 _Z_OWNED_TYPE_VALUE(_z_closure_reply_t, closure_reply)
 
-void z_closure_reply_call(const z_loaned_closure_reply_t *closure, const z_loaned_reply_t *reply);
+void z_closure_reply_call(const z_loaned_closure_reply_t *closure, z_loaned_reply_t *reply);
 
-typedef void (*z_loaned_hello_handler_t)(const z_loaned_hello_t *hello, void *arg);
```

Comment:
can we keep const for hello closure, given that it does not have any non-const functions ?